### PR TITLE
feat: add landscape orientation resources for welcome screen

### LINKED ISSUE #1044: Add landscape orientation resources for welcome screen
**Describe the feature you'd like**
Add landscape orientation resources for the welcome screen.

**Screenshots**
welcome_screen

**Additional context**
Since the app supports landscape orientation, there should be resources(layouts/strings etc) for the same. Some screens are good without needing separate resources but some are unusable without them. Welcome screen's current layout doesn't work in landscape orientation.

**Would you like to work on the issue?**
- [X] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by AnEnigmaticBug**
Fixes #1044 

Changes: This adds a landscape layout and a landscape strings.xml file. This makes the welcome screen usable in landscape configuration.

Screenshots for the change:
Before
welcome_screen
After
welcome_screen_new

### CODE REVIEW

File: app/src/main/res/layout-land/fragment_welcome.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="org.fossasia.openevent.general.WelcomeFragment">
+
+    <TextView
+        android:id="@+id/textView"
```
Comment (nikit19) [inline app/src/main/res/layout-land/fragment_welcome.xml]: Add meaningful variable names

Comment (nikit19) [reply app/src/main/res/layout-land/fragment_welcome.xml]: elsewhere too

Comment (AnEnigmaticBug) [reply app/src/main/res/layout-land/fragment_welcome.xml]: Ok. I'll add them

Comment (AnEnigmaticBug) [reply app/src/main/res/layout-land/fragment_welcome.xml]: @nikit19 the corresponding portrait layout doesn't have variable names for the same. Do I add them only in the landscape layout?

Comment (AnEnigmaticBug) [reply app/src/main/res/layout-land/fragment_welcome.xml]: If I add ids which aren't there in portrait mode layout, then someone can potentially try using them in Kotlin. Which may be error-prone. The id would exist only in landscape mode and not in portrait mode. If I should add ids, shouldn't I add them for both of them?

Comment (AnEnigmaticBug) [reply app/src/main/res/layout-land/fragment_welcome.xml]: @nikit19  I've made the changes

**Comment by iamareebjamal** [comment]
What are the differences in landscape and portrait layout?

**Comment by AnEnigmaticBug** [comment]
@iamareebjamal the landscape layout uses ConstraintLayout. This makes it have a flatter view hierarchy and allows the layout to work predictably on many devices. In the portrait layout, the globe image's size is fixed. But in the landscape layout, the image's size depends upon the space available. This prevents overlapping of UI elements.

**Comment by AnEnigmaticBug** [comment]
@nikit19 do you require any further changes?

File: app/src/main/res/layout-land/fragment_welcome.xml (hunk 1 above)
Comment (AnEnigmaticBug) [reply app/src/main/res/layout-land/fragment_welcome.xml]: @nikit19 can you please review my changes?

**Pull request merged**